service/ztests/index/query.yaml

```diff
@@ -0,0 +1,38 @@
+script: |
+  source service.sh
+  zed lake create -q test
+  zed lake use -q test@main
+  zed lake index create -q s field s
+  # Load these separately so we have 3 different objects.
+  zed lake load -q 1.zson
+  zed lake load -q 2.zson
+  zed lake load -q 3.zson
+  zed lake query -s -o /dev/null 's==127.0.0.1' 2> stats.zson
+  zed lake index update -q
+  zed lake query -s -o /dev/null 's==127.0.0.1' 2>> stats.zson
+  zed lake query -s -o /dev/null 's==1' 2>> stats.zson
+  zed lake query -s -o /dev/null 's=="hello"' 2>> stats.zson
+  zed lake query -s -o /dev/null 's=="hello" or s==1 or s==127.0.0.1' 2>> stats.zson
+  zq -z 'cut records_read, records_matched' stats.zson
```

Comment:
Any reason not to let `zed lake query` write results to standard output and stats to standard error (and then check both) like you do in `query-relative.yaml`?

Reply:
Yeah probably should do that. I didn't want to check bytes because that could change, but then again zson is pretty stable.

Reply:
Oh snap, these should be zapi as well.